sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Release History
 
 ## 1.0.0-beta.4 (Unreleased)
-- Added support for Azure Active Directory Authentication for both clients.
+### Added
+- Added support for Azure Active Directory Authentication.
+
+### Breaking Changes
+- CommunicationIdentityClient and CommunicationIdentityAsyncClient is moved to a new package, `Azure.Communication.Identity`.
```

Comment:
Probably should it be `com.azure:azure-communication-identity`  or `azure-communication-identity`

Reply:
Updated, thank you